modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/listParts.js

```diff
@@ -0,0 +1,38 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+// BEGIN
+// Fetch virtual parts.
+var result = schemaLib.listComponents({
+    application: 'myapp',
+    type: 'PART'
+});
+
+log.info('Fetched parts: ' + result.map(part => part.key).join(','));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
log.info('Fetched parts: ' + result.map((part) => part.key).join(','));
```